lib/membrane_rtc_engine/engine.ex

```diff
@@ -199,19 +199,20 @@ defmodule Membrane.RTC.Engine do
     Message,
     Track,
     Peer,
-    DisplayManager,
-    Tee
+    DisplayManager
   }
 
+  alias Membrane.RTC.Engine
+
   require Membrane.Logger
 
   @registry_name Membrane.RTC.Engine.Registry.Dispatcher
 
   @typedoc """
   RTC Engine configuration options.
 
-  `id` is used by logger. If not provided it will be generated.
-  `display_manager?` decide if `#{inspect(__MODULE__)}.DisplayManager` should be spawned for engine.
+  * `id` is used by logger. If not provided it will be generated.
+  * `display_manager?` decide if DisplayManager should be spawned for engine. It will make decision which track is sent to which peers.
```

Comment:
```suggestion
  * `id`  - this option is used by logger. If not provided it will be generated.
  * `display_manager?` - set to `true` if you want to limit number of tracks sent from `#{inspect(__MODULE__)}.Endpoint.WebRTC` to a browser.
```